lib/versionmanager.js

```diff
@@ -358,19 +359,17 @@ function queryVersions(packageMap, options = {}) {
 
     // ignore 404 errors from getPackageVersion by having them return null instead of rejecting
     function getPackageVersionProtected(dep) {
-        if (bar) {
-            bar.tick();
-        }
+        bar.tick();
```

Comment:
This is the tick that I am questioning. Does this mark any significant time? Or should we just wait for the `getPackageVersion` function to return?